### DISCUSSION TITLE
v0.47.4-20240626.190016

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#1076] [General]
* GH-1075: Fix "E1502: Lua failed to grow stack to 4001". Thanks @shanesmith for the PR!
* Dependencies updates:
    - sparkle-project/Sparkle@2.7.3
    - apple/swift-collections@1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.47.4-20240626.190016'>v0.47.4-20240626.190016</a>.</em>

**Comment by stehbar** [comment]
Hi. After upgrading to VimR 0.47.4 and skim 1.7.3 reverse search does not seem to work any longer. No idea how to debug and trace back the source of the problem. Can anybody help, please? Thanks - Stefano
